storage/namespace_test.go

```diff
@@ -344,9 +349,9 @@ func TestNamespaceRepair(t *testing.T) {
 				ChecksumDifferences: repair.NewReplicaSeriesMetadata(),
 			}
 		}
-		shard.EXPECT().Repair(testNamespaceID, repairer).Return(res, errs[i])
+		shard.EXPECT().Repair(gomock.Any(), testNamespaceID, repairTime, repairer).Return(res, errs[i])
```

Comment:
the first arg is now a context that's a bit difficult to mock since it's a local variable passed from within Repair() in `namespace.go`